tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -1031,3 +1034,53 @@ void test_file_unlink(void)
 
 	TC_PRINT("File (%s) deleted successfully!\n", TEST_FILE_RN);
 }
+
+static void *fs_api_setup(void)
+{
+	fs_register(TEST_FS_1, &temp_fs);
+	fs_mount(&test_fs_mnt_1);
+	fs_register(TEST_FS_2, &null_fs);
+	null_fs.mount = temp_fs.mount;
+	null_fs.unmount = temp_fs.unmount;
+	fs_mount(&test_fs_mnt_no_op);
+	return NULL;
+}
+
+static void fs_api_teardown(void *fixtrue)
+{
+	fs_unmount(&test_fs_mnt_no_op);
+	fs_unregister(TEST_FS_2, &null_fs);
+	null_fs.mount = null_fs.unmount = NULL;
```

Comment:
Shouldn't this line precede the `fs_register`?

Reply:
Updated.